### DISCUSSION TITLE
Official GTFS Reference links to code.google.com/p/googletransitdatafeed/wiki redirects to this repo

### DISCUSSION THREAD

**Discussion by felixguendling** [MobilityData/gtfs-validator#1313] [General]
I was interested in the fare examples:
https://code.google.com/p/googletransitdatafeed/wiki/FareExamples?hl=de

This link is contained in the GTFS reference:
https://developers.google.com/transit/gtfs/reference#fare_rulestxt

But this links to this repo (MobilityData/gtfs-validator) - which is definitely a very helpful tool (thank you very much for your work!) - but not the examples I was expecting.

Can you please help me to find the examples, I'm searching for?

**Comment by isabelle-dr** [comment]
Hello @felixguendling, here is the resource you're looking for! 
https://web.archive.org/web/20111207224351/https://code.google.com/p/googletransitdatafeed/wiki/FareExamples

PS: https://gtfs.org/schedule/reference/ contains a more up-to-date version of the spec, this link has been updated.
